### PR TITLE
Details in descriptions

### PR DESCRIPTION
**Pull request by igauravsehrawat**
3 points,
 1.links for slack,fb and june meetup 
2. target=_blank 
3.fork me strip

P.S: Don't have facebook group link.

### CODE REVIEW

**Comment by kskrishnasangeeth** [comment]
+1 for slack

File: content/pages/about.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ Chennaipy is a Python user-group, where Pythonistas in and around
 Chennai come together to share their knowledge and to contribute to
 the promotion of Python.
 
-<!-- NOTE: Upcoming event template, update and uncomment 
+ <!-- <div class="message">NOTE: Upcoming event template, update and uncomment</div> -->
 <div class="cp-info-box">
-  <b>Upcoming Events:</b> <a href="/meetup-apr-2015.html">April Meetup</a>
+  <b>Upcoming Events:</b> <a href="http://www.meetup.com/Chennaipy/events/222732435/" target="_blank">June Meetup</a>
```
Comment (shrayasr) [inline content/pages/about.md]: The June meetup still isn't announced. Lets keep this for when we announce it on the mailing list and the meetup.

File: content/pages/about.md [hunk 2]
```diff
@@ -46,8 +46,26 @@ the promotion of Python.
...
+     <a href="http://chennaipy.slack.com" target="_blank">Discuss</a>
+     </p>
+     </div>
+     <div class="pure-u-1-2 cp-grid-sect">
+     <p>
+     <i class="fa fa-facebook fa-4x" style="color:#4099FF"></i>
+     </p>
+     <p><strong>Faceboook</strong><br/>
+     <a href="https://fb.com/" target="_blank">Join</a> |
+     <a href="https://fb.com/chennaipy" target="_blank">FB posts</a>
```
Comment (shrayasr) [inline content/pages/about.md]: Please don't put a link that doesn't go anywhere. Bad idea IMO.

**Comment by shrayasr** [comment]
+1 for slack.
+1 for `target=_blank`. Long pending

**Comment by igauravsehrawat** [comment]
Ok. if there is facebook link,i will put it.

should we focus on facebook ? there is lot of college students/teenagers who are on fb, we might need to get their attention, creating fb events and stuff.

**Comment by shrayasr** [comment]
> Ok. if there is facebook link,i will put it.

Didn't you say one was already there? I don't know anything about it though. I'm not even on fb.
 
> should we focus on facebook ? there is lot of college students/teenagers who are on fb, we might need to get their attention, creating fb events and stuff.

Yes we should!

**Comment by igauravsehrawat** [comment]
removed fb, june meetup announcement
should be ready for live version !

**Comment by shrayasr** [comment]
:+1: :shipit:

**Pull request merged**